operator/src/main/resources/kafka-metrics.yaml

```diff
@@ -267,3 +267,33 @@ data:
         labels:
           quota_type: "$1"
         type: GAUGE
+      - labels:
+          clientID: $1
+        name: kafka_producer_metrics_outgoing_byte_rate
+        pattern: >-
+          kafka.producer<type=producer-metrics, client-id=(.+)><>outgoing-byte-rate
+        type: GAUGE
+      - labels:
+          clientID: $1
+        name: kafka_producer_metrics_$2
+        pattern: >-
+          kafka.producer<type=producer-metrics, client-id=(.+)><>(connection-.+):
```

Comment:
> Trailing semicolon here is to avoid capturing the value along with the attribute name

I guess that's down to REs greedy nature. 

I'd make the `.` in the second pattern space more discerning (perhaps `[\w-]+[\w]` or even enumerate the names you want.

What's the domain of $2? I think it is `connection-close-rate`, `connection-close-total` etc described by 
https://kafka.apache.org/documentation/#selector_monitoring

Reply:
Yes, the 4 listed on that page, plus also `connection-count` (which is the one I've proposed that we us in the alert)